decred/tests/unit/examples/test_examples.py

```diff
@@ -0,0 +1,28 @@
+"""
+Copyright (c) 2019, The Decred developers
+"""
+
+import importlib.util
+import os
+from pathlib import Path
+import py_compile
+from tempfile import TemporaryDirectory
+
+import decred
+
+
+def test_compile():
+    exampleDir = Path(os.path.realpath(decred.__file__)).parent.parent / "examples"
+
+    with TemporaryDirectory() as tempDir:
+        for filename in os.listdir(exampleDir):
+            # The "plot_ticket_price.py" file imports matplotlib, which is not a TD
+            # dependency, so skip that file.
+            if not filename.endswith(".py") or "plot_" in filename:
+                continue
+            path = os.path.join(exampleDir, filename)
```

Comment:
Similarly I'd use `pathlib` syntax here. From the docs it's not clear whether `py_compile` and `importlib` accept Path objects: if not, just wrap them in a `str()` call.